src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js

```diff
@@ -77,8 +90,38 @@ const MappingProfilesContainer = ({
 
 MappingProfilesContainer.propTypes = {
   match: matchShape.isRequired,
+  mutator: PropTypes.shape({
+    mappingProfiles: PropTypes.shape({
+      GET: PropTypes.func.isRequired,
+    }),
+  }),
   history: historyShape.isRequired,
   location: locationShape.isRequired,
 };
 
-export default MappingProfilesContainer;
+MappingProfilesContainer.manifest = Object.freeze({
+  initializedFilterConfig: { initialValue: false },
+  query: { initialValue: {} },
+  resultCount: { initialValue: 5000 },
```

Comment:
Could you please create constant INITIAL_RESULT_COUNT in constants with 100 per request. The overall majority of apps are using 30.